src/test/mochitest/examples/simple1.js

```diff
@@ -7,33 +7,25 @@ function main() {
 }
 
 function doEval() {
```

Comment:
Some of these files seem to use source maps, are we sure it's correct to rewrite them?